pkg/workflow_template.go

```diff
@@ -334,6 +388,22 @@ func (c *Client) listWorkflowTemplateVersions(namespace, uid string) (workflowTe
 	return
 }
 
+func (c *Client) listWorkflowTemplateVersionsDBAll(paginator *pagination.PaginationRequest) (workflowTemplateVersions []*WorkflowTemplateVersion, err error) {
+	return c.selectAllWorkflowTemplateVersionsDB(paginator)
+}
```

Comment:
@aleksandrmelnikov Do we need this method? Why not just call 
```golang
selectAllWorkflowTemplateVersionsDB
```

Reply:
Agreed